### PR TITLE
staticcheck:remove duplicate import packages

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Delete a package imported twice, for example:
```shell
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

### CODE REVIEW

File: pkg/apis/kops/channel.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 
 	"github.com/blang/semver"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (tanjunchen) [inline pkg/apis/kops/channel.go]: good catch

**Comment by tanjunchen** [comment]
/retest
ref:https://github.com/kubernetes/kops/issues/7800

**Comment by hakman** [comment]
@yuxiaobo96 please try and run `make gofmt`. It should fix the formatting.

**Comment by yuxiaobo96** [comment]
@hakman 
Thanks for your advice, I have a question.
When I run `./hack/verify-gofmt.sh`:
```shell
xargs: bazel: without that file or directory
```
I don't understand why.

**Comment by hakman** [comment]
Do you have `bazel` installed?

**Comment by yuxiaobo96** [comment]
@hakman 
Thank you, I will install in a while.

File: pkg/client/simple/vfsclientset/instancegroup.go [hunk 2]
```diff
@@ -27,7 +27,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/klog"
-	"k8s.io/kops/pkg/apis/kops"
```
Comment (tanjunchen) [inline pkg/client/simple/vfsclientset/instancegroup.go]: kopsapi "k8s.io/kops/pkg/apis/kops" i see the alias in the other packages , so recommend kopsapi  i think , but it is ok.

Comment (rifelpet) [reply pkg/client/simple/vfsclientset/instancegroup.go]: I agree, `api` is too vague so if we switched the `k8s.io/kops/pkg/apis/kops` import in this file and `cloud_instance_group.go` to use `kopsapi` that could be more informative.

Comment (yuxiaobo96) [reply pkg/client/simple/vfsclientset/instancegroup.go]: I am going to update.

File: pkg/apis/kops/channel.go [hunk 3]
```diff
@@ -36,7 +35,7 @@ const (
 )
 
 type Channel struct {
-	v1.TypeMeta `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
 	ObjectMeta  metav1.ObjectMeta `json:"metadata,omitempty"`
```
Comment (hakman) [inline pkg/apis/kops/channel.go]: This is the reason `verify-gofmt` fails, can you update?
```suggestion
	ObjectMeta      metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment (yuxiaobo96) [reply pkg/apis/kops/channel.go]: You are great.

Comment (hakman) [reply pkg/apis/kops/channel.go]: Np. 5 spaces to align with `metav1.TypeMeta`.

Comment (hakman) [reply pkg/apis/kops/channel.go]: OK, tests are passing now. Please squash the commits and force push.

Comment (yuxiaobo96) [reply pkg/apis/kops/channel.go]: OK.

**Comment by rifelpet** [comment]
Thanks for the improvement!
/approve
/lgtm

**Pull request merged**